opal/mca/btl/tcp/btl_tcp_endpoint.c

```diff
@@ -483,10 +483,6 @@ static void *mca_btl_tcp_endpoint_complete_accept(int fd, int flags, void *conte
         mca_btl_tcp_endpoint_event_init(btl_endpoint);
         MCA_BTL_TCP_ENDPOINT_DUMP(10, btl_endpoint, true, "event_add(recv) [endpoint_accept]");
         opal_event_add(&btl_endpoint->endpoint_recv_event, 0);
-        if (mca_btl_tcp_event_base == opal_sync_event_base) {
```

Comment:
I think this code is still needed for the TCP BTL.  Unlike the runtime, the TCP BTL may use the primary event base (saving a bunch of thread context switch time.

Reply:
sure, will undo.

Reply:
Repushed, thanks.